middlewares/article.js

```diff
@@ -32,5 +35,48 @@ class ArticleMiddleware {
     }
     next();
   }
+
+  /**
+   * Verify if article exists
+   * @param  {object} req - Request object
+   * @param {object} res - Response object
+   * @param {function} next - calls the next middleware
+   * @return {json} Returns json object
+   * @static
+   */
+  static async isArticleExist(req, res, next) {
+    const { articleId } = req.body;
+    if (Number.isNaN(Number(articleId))) {
+      return res.status(400).json({
+        status: 400,
+        errors: { body: ['Invalid data'] }
+      });
+    }
+    const article = await Article.findOne({ where: { id: articleId }, logging: false });
```

Comment:
Why are you still having Article.findOne here? is this your article helper class